pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
     "numpy",
     "pandas",
     "rich",
-    "fancylog",
+    "h5py",
 ]
```

Comment:
I would add either a `tables` dependency or instructions of how to install e.g. from conda. By default I get `ImportError: Missing optional dependency 'pytables'.  Use pip or conda to install pytables.` when runjning the tests.

Reply:
Thanks. I added pytables here, and will add installation isntructions to the README

Reply:
I added conda-related instructions to the README